### PR TITLE
Add user group stats page, user stats page, and logged-in user homepage ADRs

### PR DESCRIPTION
**Pull request by mcbouslog**
## Package
- docs

## Describe your changes
- add ADR 51: user group stats page
- add ADR 52: user stats page
- add ADR 53: logged-in user homepage

## How to Review
_Helpful explanations that will make your reviewer happy:_
- What user actions should my reviewer step through to review this PR?
- Are there plans for follow up PR’s to further fix this bug or develop this feature?
  - ADRs for user library and root app

# Checklist
_PR Creator - Please cater the checklist to fit the review needed for your code changes._
_PR Reviewer - Use the checklist during your review. Each point should be checkmarked or discussed before PR approval._

## General
- [ ] Tests are passing locally and on Github
- [ ] Documentation is up to date and changelog has been updated if appropriate

### CODE REVIEW

File: docs/arch/adr-51.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+Create export member statistics to CSV functionality for applicable user groups.
+
+## Status
+
+Proposed
+
+## Consequences
+
+Create client, components, and features noted.Create user group stats page in new library `lib-user` per [FEM packages structure discussion](https://github.com/zooniverse/front-end-monorepo/discussions/5089).
```
Comment (goplayoutside3) [inline docs/arch/adr-51.md:37]:
```suggestion
Create client, components, and features noted. Create user group stats page in new library `lib-user` per [FEM packages structure discussion](https://github.com/zooniverse/front-end-monorepo/discussions/5089).
```

File: docs/arch/adr-53.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+The logged-in user homepage will list recent projects, classifications, and user groups.
+
+## Decision
+
+Create ERAS client.
+
+Create shared stats components in lib-react-components.
+
+Refactor profile ribbon from Panoptes project_preferences request(s) to ERAS user stats request.
```
Comment (goplayoutside3) [inline docs/arch/adr-53.md:21]:
```suggestion
Refactor profile ribbon from Panoptes `project_preferences` request(s) to ERAS user stats request.
```

File: docs/arch/adr-51.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+## Context
+
+We will create a user group stats page to provide statistical insights and data visualization of a user group’s Zooniverse activity. The user group stats page will utilize the new Zooniverse stats service - [ERAS](https://github.com/zooniverse/eras) (Enhanced Running Average Stats Service) - in addition to existing [Panoptes](https://github.com/zooniverse/panoptes) resources (notably `users`, `user_groups`, and `projects`).
+
+### Features
+
+The user group stats page will show statistical information based on user group member role and user group `stats_visibility` property.
+
+Statistical information will include a bar chart of classification counts or hours - changeable by period and filterable by project, top projects, and could include top individual contributors and all user group members’ statistics.
```
Comment (goplayoutside3) [inline docs/arch/adr-51.md:13]: I initially read this sentence as if the bar chart is filterable by top projects too so just a NB suggestion to phrase it as "a bar chart of classification counts or hours (changeable by period and filterable by project,) top projects, etc".

**Review by goplayoutside3 — APPROVED** [review]
Looks good! Just a couple of NB suggestions.

**Comment by goplayoutside3** [comment]
@mcbouslog just checking in to see if this PR will be merged soon? I wrte ADR 54 about app-root in https://github.com/zooniverse/front-end-monorepo/pull/5423

File: docs/arch/adr-51.md (hunk 3 above)
Comment (mcbouslog) [inline docs/arch/adr-51.md:13]:
```suggestion
Statistical information will include a bar chart of classification counts or hours (changeable by period and filterable by project), top projects, and could include top individual contributors and all user group members’ statistics.
```

**Comment by mcbouslog** [comment]
> @mcbouslog just checking in to see if this PR will be merged soon? I wrte ADR 54 about app-root in #5423

Yes, I'll merge today.

**Pull request merged**